statsproduct.php

```diff
@@ -443,29 +443,10 @@ protected function getData($layers)
             $this->setDateGraph($layers, true);
         } else {
             $product = new Product($this->id_product, false, (int)$this->getLang());
-
-            $comb_array = array();
-            $assoc_names = array();
-            $combinations = $product->getAttributeCombinations((int)$this->getLang());
-            foreach ($combinations as $combination) {
-                $comb_array[$combination['id_product_attribute']][] = array(
-                    'group' => $combination['group_name'],
-                    'attr' => $combination['attribute_name']
-                );
-            }
-            foreach ($comb_array as $id_product_attribute => $product_attribute) {
-                $list = '';
-                foreach ($product_attribute as $attribute) {
-                    $list .= trim($attribute['group']).' - '.trim($attribute['attr']).', ';
-                }
-                $list = rtrim($list, ', ');
-                $assoc_names[$id_product_attribute] = $list;
-            }
-
-            $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS($this->query);
+	    $combinationName = str_replace($product->name, '', $row['product_name']);
             foreach ($result as $row) {
```

Comment:
I think something's wrong here 🤔
You're using `$row` before the loop and use `$result` for the loop but you deleted it just before 😅

Reply:
😱